### PR TITLE
[aarch64] Add build wheel script for NVPL for CUDA ARM

### PR DESCRIPTION
**Pull request by tinglvv**
docker change - https://github.com/pytorch/pytorch/pull/132811

cc @atalman @Aidyn-A @ptrblck @nWEIdia

### CODE REVIEW

File: aarch64_linux/aarch64_wheel_ci_build.py [hunk 1]
```diff
@@ -86,7 +86,10 @@ def update_wheel(wheel_path) -> None:
         "/usr/local/cuda/lib64/libcudnn_engines_precompiled.so.9",
         "/usr/local/cuda/lib64/libcudnn_heuristic.so.9",
         "/opt/conda/envs/aarch64_env/lib/libgomp.so.1",
-        "/opt/OpenBLAS/lib/libopenblas.so.0",
```
Comment (Aidyn-A) [inline aarch64_linux/aarch64_wheel_ci_build.py:89]: Is this file used for CUDA-only build? My gut feels like removing `/opt/OpenBLAS/lib/libopenblas.so.0` here will cause missing libraries in CPU-only build.

Comment (tinglvv) [reply aarch64_linux/aarch64_wheel_ci_build.py:89]: thanks for catching this, let me add the guards to here as well

File: aarch64_linux/aarch64_wheel_ci_build.py [hunk 2]
```diff
@@ -201,9 +211,11 @@ def parse_arguments():
             "LD_LIBRARY_PATH=/pytorch/build/lib:/acl/build:$LD_LIBRARY_PATH "
             "ACL_INCLUDE_DIR=/acl/build "
             "ACL_LIBRARY=/acl/build "
-            "BLAS=OpenBLAS "
-            "OpenBLAS_HOME=/OpenBLAS "
         )
+        if enable_cuda:
+            build_vars += "BLAS=NVPL "
+        else:
+            build_vars += "BLAS=OpenBLAS " "OpenBLAS_HOME=/OpenBLAS "
```
Comment (Aidyn-A) [inline aarch64_linux/aarch64_wheel_ci_build.py:218]:
```suggestion
            build_vars += "BLAS=OpenBLAS OpenBLAS_HOME=/OpenBLAS "
```

**Review by Aidyn-A — APPROVED** [review]
Thanks! Looks good to me now. I have only a minor cosmetic suggestion left below. Can it be verified in https://github.com/pytorch/pytorch/pull/132811?

**Comment by tinglvv** [comment]
> Thanks! Looks good to me now. I have only a minor cosmetic suggestion left below. Can it be verified in [pytorch/pytorch#132811](https://github.com/pytorch/pytorch/pull/132811)?

Thanks! will have to work with @atalman to merge https://github.com/pytorch/pytorch/pull/132811 first and then merge this one right after.

**Review by atalman — APPROVED** [review]
lgtm

**Comment by tinglvv** [comment]
Merging as https://github.com/pytorch/pytorch/pull/132811 is merged

**Pull request merged**